src/main/java/carpet/script/api/WorldAccess.java

```diff
@@ -776,12 +777,18 @@ else if (args.get(0) instanceof MapValue)
 
             if (sourceBlockState == targetBlockState && data == null)
                 return Value.FALSE;
+
```

Comment:
what are these spaces for?

Reply:
If I'm not mistaken, those are places where I had code for debugging, but didn't bother or forgot to remove the line breaks when I removed the code, since it seemed like a natural place to have them.